src/mono/mono/containers/dn-rt-utils.c

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Shouldn't the naming of this file follow the naming of the header, dn-rt-mono.c?